common/src/main/java/tachyon/exception/TachyonException.java

```diff
@@ -57,19 +54,47 @@ public TachyonTException toTachyonTException() {
     return new TachyonTException(mType.name(), getMessage());
   }
 
+  /**
+   * Construct a TachyonException from a TachyonTException.
```

Comment:
`Constructs` (use 3rd person not 2nd person as per http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#styleguide)